demos/python_demos/deep_speech_demo/deep_speech_demo.py

```diff
@@ -0,0 +1,195 @@
+"""
+from __future__ import print_function
+import sys
+import os
+from argparse import ArgumentParser, SUPPRESS
+import numpy as np
+import logging as log
+from time import time
+from openvino.inference_engine import IENetwork, IECore
+
+# For Speech Feature
+import wave, struct
+import codecs
+
+from speech_features import audio_spectrogram, mfcc 
+from ctc_beamsearch_decoder import ctc_beam_search_decoder
+
+
+def build_argparser():
+    parser = ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
+    args.add_argument("-m", "--model", help="Required. Path to an .xml file with a trained model.", required=True,
+                      type=str)
+    args.add_argument("-i", "--input", help="Required. Path to an audio file.",
+                      required=True,
+                      type=str)
+    args.add_argument("-l", "--cpu_extension",
+                      help="Optional. Required for CPU custom layers. "
+                           "MKLDNN (CPU)-targeted custom layers. Absolute path to a shared library with the"
+                           " kernels implementations.", type=str, default=None)
+    args.add_argument("-d", "--device",
+                      help="Optional. Specify the target device to infer on; CPU, GPU, FPGA, HDDL, MYRIAD or HETERO: is "
+                           "acceptable. The sample will look for a suitable plugin for device specified. Default "
+                           "value is CPU",
+                      default="CPU", type=str)
+    args.add_argument("-a", "--alphabet", help="Required. Path to a alphabet file.", required=True, default="alphabet_b.txt", type=str)
+
+    return parser
+
+class Alphabet(object):
+    def __init__(self, config_file):
+        self._label_to_str = []
+        self._str_to_label = {}
+        self._size = 0
+        with codecs.open(config_file, 'r', 'utf-8') as fin:
+            for line in fin:
+                if line[0:2] == '\\#':
+                    line = '#\n'
+                elif line[0] == '#':
+                    continue
+                self._label_to_str += line[:-1] # remove the line ending
+                self._str_to_label[line[:-1]] = self._size
+                self._size += 1
+
+    def string_from_label(self, label):
+        return self._label_to_str[label]
+
+    def label_from_string(self, string):
+        return self._str_to_label[string]
+
+    def size(self):
+        return self._size
+
+n_input    = 26
+n_context  = 9
+n_steps    = 16
+numcep     = n_input
+numcontext = n_context
+beamwidth  = 10
+
+def main():
+    log.basicConfig(format="[ %(levelname)s ] %(message)s", level=log.INFO, stream=sys.stdout)
+    args = build_argparser().parse_args()
+    
+    alphabet = Alphabet(os.path.abspath(args.alphabet))
+    
+    # Speech feature extration
+    _wave = wave.open('1272-128104-0000.wav', 'rb')
```

Comment:
I think you forgot to use the `-i` option here. Also, please use `with` to ensure the file is closed.